### PR TITLE
[key selector] Load default config and adjust timeout

### PR DESCRIPTION
**Pull request by kotori2**

### CODE REVIEW

File: key-selector/src/main/cpp/key_selector.cpp [hunk 1]
```diff
@@ -231,20 +231,24 @@ uint32_t get_event() {
     }
 }
 
+// for phone which has no button
+uint16_t timeout = 10;
+Languages* l = nullptr;
```
Comment (yujincheng08) [inline key-selector/src/main/cpp/key_selector.cpp:236]: std::unique_ptr

**Review by yujincheng08 — CHANGES_REQUESTED** [review]

**Review by yujincheng08 — CHANGES_REQUESTED** [review]
unique!! not shared!!

**Comment by kotori2** [comment]
> unique!! not shared!!

くさ

**Review by yujincheng08 — APPROVED** [review]
很有精神！

**Pull request merged**